src/NotificationBar/NotificationBar.styles.ts

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from 'styled-components';
+import { NotificationIcon, UpgradeIcon } from '../Icons';
+import { Props } from './NotificationBar';
+
+export const BarDiv = styled.div<Props>`
```

Comment:
The text in the example doesn't seem to be aligned in the horizontal centre.

Reply:
In the designs from silvia, the text shoudnt be centered in the page anyway

Reply:
durp, verticle centre.